app/src/main/java/com/example/safehopper/MainActivity.java

```diff
@@ -7,7 +7,13 @@
 import androidx.navigation.ui.AppBarConfiguration;
 import androidx.navigation.ui.NavigationUI;
 
+import com.example.safehopper.models.Contact;
```

Comment:
Please remove the Contact, User, and GSON imports as they won't be necessary for the app to run. They were just for testing.